README.md

```diff
@@ -1,43 +1,33 @@
 # op-ui-2
 
-> A Vue.js project
+## Up and Running
+
+1. run `npm ci`
+2. in your browser, open `localhost:8080`
 
 ## Build Setup
 
-``` bash
-# install dependencies
-npm install
+For a detailed explanation on how things work, check out the [guide](http://vuejs-templates.github.io/webpack/) and [docs for vue-loader](http://vuejs.github.io/vue-loader).
 
-# serve with hot reload at localhost:8080
-npm run dev
+# Configuration
 
-# build for production with minification
-npm run build
+## Environment variables
 
-# build for production and view the bundle analyzer report
-npm run build --report
+Inside the root folder, create a .env with the following variables:
+(note: a quirk is that all strings in the .env need to be enclosed like this '"STRING_HERE"')
 
-# run unit tests
-npm run unit
+```
 
-# run e2e tests
-npm run e2e
+PROD_GOOGLE_API_KEY:
+PROD_FIREBASE_DATABASE_URL:
+PROD_FIREBASE_PROJECT_ID:
+PROD_FIREBASE_STORAGE_BUCKET:
+PROD_FIREBASE_MESSAGING_SENDER_ID:
 
-# run all tests
-npm test
 ```
 
-For a detailed explanation on how things work, check out the [guide](http://vuejs-templates.github.io/webpack/) and [docs for vue-loader](http://vuejs.github.io/vue-loader).
+todo: use `DEV_` variables when in dev mode (set this in config/dev.env.js). Currently all the variables names are `PROD_`
```

Comment:
@weiweitoo actually right now I'm a bit confused myself lol. our current implementation is using .env whereas vue by default provides us the ability to have environment-specific env variables. I was trying to say we should have environment specific env variables. I need to do a revision of how this works in vue.